src/backend/access/aocs/aocssegfiles.c

```diff
@@ -100,9 +100,14 @@ InsertInitialAOCSFileSegInfo(Relation prel, int32 segno, int32 nvp)
 
 	segrel = heap_open(prel->rd_appendonly->segrelid, RowExclusiveLock);
 
-	InsertFastSequenceEntry(prel->rd_appendonly->segrelid,
-							(int64) segno,
-							0);
+	/*
+	 * Create a new entry in gp_fastsequence or increment
+	 * an existing one (it can exist after transaction rollback).
+	 */
+	GetFastSequences(prel->rd_appendonly->segrelid,
+					 (int64) segno,
+					 0,
+					 NUM_FAST_SEQUENCES);
```

Comment:
Why do we set `last_sequence` to `NUM_FAST_SEQUENCES` here?

Reply:
A good question in fact - there is no sense in it and if we use `0` instead of `NUM_FAST_SEQUENCES` we don't change row numeration in segment files. Thanks, @leskin-in ! I have again requested your review.